packages/components/src/components/DataAnalytics.tsx

```diff
@@ -42,7 +41,7 @@ const Label = styled.span`
     display: flex;
 `;
 
-const Heading = styled(H2)`
+const Heading = styled.h2`
```

Comment:
Why not to use our wrapper?

Reply:
Because its style is not used anyways, it's rewritten to something else. Maybe H3 would fit, just thought about it... We can improve it later once we get to the welcome page.